apps/mark/frontend/test/helpers/mock_api_client.tsx

```diff
@@ -193,6 +204,36 @@ export function createApiMock() {
     expectEjectUsbDrive() {
       mockApiClient.ejectUsbDrive.expectCallWith().resolves();
     },
+
+    expectGetElectionState(electionState?: Partial<ElectionState>) {
+      electionStateRef.current = electionState
+        ? {
+            ...electionStateRef.current,
+            ...electionState,
+          }
+        : initialElectionState;
+      mockApiClient.getElectionState
+        .expectCallWith()
+        .resolves(electionStateRef.current);
+    },
```

Comment:
To make testing easier on myself, I gave `apiMock` some state so you only have to expect the diffs.